README.md

```diff
@@ -1 +1,27 @@
-# hello-microverse-project-
\ No newline at end of file
+# hello-microverse-project-
+
+> This project was made with the aim of following GitHub Flow, write descriptive messages and descriptions.
+
+
+## Built With
+
+- HTML
+- CSS
+- linter
+
+
+
+## Authors
+
+👤 **David Tamayo**
+
+- GitHub: [@IngDavidTM](https://github.com/IngDavidTM)
+- Twitter: [@David5TM](https://twitter.com/David5TM)
+
+## 🤝 Contributing
```

Comment:
- [ ] You have done well updating your readme 😄, but I think you should improve on your readme by following the [ template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse. Please ensure your readme looks like the template provided 👍🏼 .